dist2src/worker/updater.py

```diff
@@ -2,16 +2,15 @@
 # SPDX-License-Identifier: MIT
 
 import os
-import re
-
 from logging import getLogger
 from typing import List, Optional, Tuple
+
+from gitlab import GitlabGetError
+from ogr.services.gitlab import GitlabProject
 from ogr.services.pagure import PagureProject
```

Comment:
For some reason I was expecting `PagureProject` to disappear, but now I realise, that the dist-git repos for c8s were not move to GitLab yet. Do you know if this is going to happen in the future or note?

Reply:
well, I was told they could but we're at least 6 months away from that - it was said without any commitment so we're stuck with Pagure here

Reply:
ack